backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/nodes/PackageNodeDescriptionProvider.java

```diff
@@ -169,12 +162,11 @@ private DropNodeTool createDropFromDiagramTool(IViewDiagramElementFinder cache)
     private NodeToolSection createNodeToolSection(IViewDiagramElementFinder cache) {
         var nodeTools = new ArrayList<NodeTool>();
 
-        GeneralViewDiagramDescriptionProvider.DEFINITIONS.forEach(definition -> {
-            nodeTools.add(this.createNodeTool(cache.getNodeDescription(GVDescriptionNameGenerator.getNodeName(definition)).get(), definition));
-        });
-
-        GeneralViewDiagramDescriptionProvider.USAGES.forEach(usage -> {
-            nodeTools.add(this.createNodeTool(cache.getNodeDescription(GVDescriptionNameGenerator.getNodeName(usage)).get(), usage));
+        GeneralViewDiagramDescriptionProvider.TOOL_SECTIONS.forEach((sectionName, elements) -> {
+            elements.forEach(element -> {
+                var optNodeDescription = cache.getNodeDescription(GVDescriptionNameGenerator.getNodeName(element));
+                nodeTools.add(this.createNodeTool(optNodeDescription.get(), element));                
+            });
         });
 
         nodeTools.add(this.createNodeTool(cache.getNodeDescription(PackageNodeDescriptionProvider.NAME).get(), SysmlPackage.eINSTANCE.getPackage()));
```

Comment:
You use TOOL_SECTIONS above to create add the tools needed but forget to remove this line.
But Package is already handled by the TOOL_SECTIONS.

Reply:
Done